mismi-core/src/Mismi/Environment.hs

```diff
@@ -40,5 +49,13 @@ getRegionFromEnv = do
     (either (throwM . RegionUnknown . T.pack) return . fromText . T.pack)
     mr
 
+discoverAwsEnv :: IO (Either EnvError Env)
+discoverAwsEnv = runEitherT $ do
+  r <- EitherT . fmap (maybeToRight MissingRegion) $ getRegionFromEnv
+  lift $ newEnv r Discover
```

Comment:
Not sure if this was on purpose, but you have omitted the `AWS_SECURITY_TOKEN` [lookup](https://github.com/ambiata/mismi/pull/188/files#diff-4bf4d01490c6909528be1107308d244fL76). Do we not want to expose this anymore? (this was used to `sts` operations with these runners previously)